src/Components/HomeCards/HomeCards.tsx

```diff
@@ -0,0 +1,66 @@
+import React from "react";
+import styles from "./HomeCards.module.css";
+
+function HomeCards() {
+  return (
+    <article className={styles.cards}>
```

Comment:
I'd consider another element here than `article`. 
["The `article` HTML element represents a self-contained composition in a document, page, application, or site, which is intended to be independently distributable or reusable (e.g., in syndication)."](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/article)